src/Components/Search/NewSearch/NewSearchBarInput.tsx

```diff
@@ -0,0 +1,231 @@
+import { AutocompleteInput, Box, BoxProps } from "@artsy/palette"
+import { SearchInputContainer } from "Components/Search/SearchInputContainer"
+import { Router } from "found"
+import { ChangeEvent, FC, useContext, useState } from "react"
+import { useTranslation } from "react-i18next"
+import { createRefetchContainer, graphql, RelayRefetchProp } from "react-relay"
+import { SystemQueryRenderer } from "System/Relay/SystemQueryRenderer"
+import {
+  SystemContext,
+  SystemContextProps,
+  withSystemContext,
+} from "System/SystemContext"
+import { extractNodes } from "Utils/extractNodes"
+import { useDidMount } from "Utils/Hooks/useDidMount"
+import { NewSearchBarInput_viewer$data } from "__generated__/NewSearchBarInput_viewer.graphql"
+import { NewSearchBarInputSuggestQuery } from "__generated__/NewSearchBarInputSuggestQuery.graphql"
+import createLogger from "Utils/logger"
+import { NewSearchInputPillsFragmentContainer } from "Components/Search/NewSearch/NewSearchInputPills"
+import { SuggestionItem } from "Components/Search/Suggestions/SuggestionItem"
+import { NewSearchBarFooter } from "Components/Search/NewSearch/NewSearchBarFooter"
+
+const logger = createLogger("Components/Search/NewSearchBar")
+
+export interface NewSearchBarInputProps extends SystemContextProps {
+  relay: RelayRefetchProp
+  router?: Router
+  viewer: NewSearchBarInput_viewer$data
+  isXs: boolean
+}
+
+const NewSearchBarInput: FC<NewSearchBarInputProps> = ({
+  isXs,
+  relay,
+  router,
```

Comment:
do we need it? If not, then we can remove it and `withSystemContext` [here](https://github.com/artsy/force/blob/32245276e87a474e36b1ed957608d2a7a30e9a26/src/Components/Search/NewSearch/NewSearchBarInput.tsx#L114)

Reply:
removed `router` for now, but I think we will need `withSystemContext` if we want to replicate [old search behaviour](https://github.com/artsy/force/blob/main/src/Components/Search/SearchBar.tsx#L481)

Reply:
> we will need `withSystemContext`

Yes, that's about what I thought. But it is better to use `useRouter` (as far as I understand, we will need `route`) instead of `withSystemContext` and for this reason I suggested to delete it for now